doc/source/api.rst

```diff
@@ -5,66 +5,108 @@
 API Reference
 =============
 
-.. currentmodule:: cvpy
+**********
+Annotation
+**********
 
-Biomedical Images
------------------
+Base
+====
+
+Credentials
+-----------
+
+.. currentmodule:: cvpy.annotation.base.Credentials
 
-.. currentmodule:: cvpy.biomedimage.BiomedImage
-    
 .. autosummary::
     :toctree: generated/
 
-    BiomedImage
-    BiomedImage.quantify_sphericity
-    BiomedImage.mask_image
-    BiomedImage.morphological_gradient
+    Credentials
 
-CAS Thread Tuner
-----------------
+Project
```

Comment:
Since Project is a base class, we don't expect end users to ever create or interact with this class. So, we can completely remove it from the documentation.